src/SConscript

```diff
@@ -217,4 +219,4 @@ if GetOption('enable_pulseaudio_modules'):
         env.AddDistFile(env['ROC_PULSE_MODULEDIR'], target)
 
     env.Alias('pulse', targets, env.Action(''))
-    env.AlwaysBuild('pulse')
+    env.AlwaysBuild('pulse')
```

Comment:
Nitpick: missing trailing newline.

Hint: you can detect such space-only changes by reviewing your changes on GitHub web interface after pushing them. It highlights them.